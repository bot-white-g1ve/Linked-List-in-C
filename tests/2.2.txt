NEW 1
a
DELETE 0 0
DELETE 0 0
NEW 2
a
b
DELETE 1 -1