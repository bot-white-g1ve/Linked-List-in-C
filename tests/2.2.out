List 0: a
List 0: 
INVALID COMMAND: DELETE
List 1: a -> b
List 1: a
